internal/plugins/ansible/scaffolds/init.go

```diff
@@ -0,0 +1,110 @@
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package scaffolds
+
+import (
+	"fmt"
+
+	"sigs.k8s.io/kubebuilder/pkg/model"
+	"sigs.k8s.io/kubebuilder/pkg/model/config"
+	"sigs.k8s.io/kubebuilder/pkg/plugin/scaffold"
+
+	"github.com/operator-framework/operator-sdk/internal/plugins/ansible/templates"
+	"github.com/operator-framework/operator-sdk/internal/plugins/ansible/templates/config/kustomize"
+	"github.com/operator-framework/operator-sdk/internal/plugins/ansible/templates/config/manager"
+	"github.com/operator-framework/operator-sdk/internal/plugins/ansible/templates/config/prometheus"
+	"github.com/operator-framework/operator-sdk/internal/plugins/ansible/templates/config/rbac"
+
+	"github.com/operator-framework/operator-sdk/internal/kubebuilder/machinery"
+)
+
+const (
+	// KustomizeVersion is the kubernetes-sigs/kustomize version to be used in the project
+	KustomizeVersion = "v3.5.4"
+
+	imageName = "controller:latest"
+)
+
+var _ scaffold.Scaffolder = &initScaffolder{}
+
+type initScaffolder struct {
+	config           *config.Config
+	apiScaffolder    scaffold.Scaffolder
+	generatePlaybook bool
+}
+
+// NewInitScaffolder returns a new Scaffolder for project initialization operations
+func NewInitScaffolder(config *config.Config, apiScaffolder scaffold.Scaffolder, generatePlaybook bool) scaffold.Scaffolder {
+	return &initScaffolder{
+		config:           config,
+		apiScaffolder:    apiScaffolder,
+		generatePlaybook: generatePlaybook,
+	}
+}
+
+func (s *initScaffolder) newUniverse() *model.Universe {
+	return model.NewUniverse(
+		model.WithConfig(s.config),
+	)
+}
+
+// Scaffold implements Scaffolder
+func (s *initScaffolder) Scaffold() error {
+	switch {
+	case s.config.IsV3():
+		if err := s.scaffold(); err != nil {
+			return err
+		}
+		if s.apiScaffolder != nil {
+			return s.apiScaffolder.Scaffold()
+		}
+		return nil
+	default:
+		return fmt.Errorf("unknown project version %v", s.config.Version)
+	}
+}
+
+func (s *initScaffolder) scaffold() error {
+	return machinery.NewScaffold().Execute(
+		s.newUniverse(),
+		&templates.Dockerfile{GeneratePlaybook: s.generatePlaybook},
+		&templates.RequirementsYml{},
+		&templates.Watches{},
+
+		&rbac.KustomizeRBAC{},
+		&rbac.ClientClusterRole{},
+		&rbac.AuthProxyRole{},
+		&rbac.AuthProxyRoleBinding{},
+		&rbac.AuthProxyService{},
+		&rbac.LeaderElectionRole{},
+		&rbac.LeaderElectionRoleBinding{},
+		&rbac.Role{},
+		&rbac.RoleBinding{},
+
+		&prometheus.Kustomization{},
+		&prometheus.ServiceMonitor{},
+
+		&manager.Manager{Image: imageName},
+		&manager.Kustomization{},
+
+		&kustomize.Kustomize{},
+		&kustomize.AuthProxyPatch{},
+
+		// TODO(asmacdo) audit makefile
+		&templates.Makefile{},
```

Comment:
same here.

Reply:
```go 
	return machinery.NewScaffold().Execute(
		s.newUniverse(),
		&templates.GitIgnore{},
		&templates.AuthProxyRole{},
		&templates.AuthProxyRoleBinding{},
		&metricsauth.AuthProxyPatch{},
		&metricsauth.AuthProxyService{},
		&metricsauth.ClientClusterRole{},
		&manager.Config{Image: imageName},
		&templates.Makefile{
			Image:               imageName,
			KustomizeVersion:    KustomizeVersion,
			AnsibleOperatorVersion: AnsibleOperatorVersion,
		},
		&templates.Dockerfile{
			AnsibleOperatorVersion: AnsibleOperatorVersion,
		},
		&templates.Kustomize{},
		&templates.ManagerRoleBinding{},
		&templates.LeaderElectionRole{},
		&templates.LeaderElectionRoleBinding{},
		&templates.KustomizeRBAC{},
		&templates.Watches{},
		&manager.Kustomization{},
		&prometheus.Kustomization{},
		&prometheus.ServiceMonitor{},
                // specifics for Ansible
                &templates.RequirementsYml{}, 
                 
	)
}

```

Hi, @fabianvf  @asmacdo, 

By comparing it with Helm I just see 1 diff:  `  &templates.RequirementsYml{}, ` 
Also, see that Makefile and dockerfile require we pass the same value as consts as well.

Reply:
Actually it is missing more one specific template for Ansible which is to scaffold the file `.travis.yml` in the root dir.

Reply:
We are deliberately not scaffolding `.travis.yml` anymore, it is too opinionated, but that file can be added to the docs. molecule dir is now finished.